### ISSUE TITLE
An argument cotaining spaces passed to an alias is split

### ISSUE THREAD

**Issue by tiibun** [pyprojectx/pyprojectx#16]
**Describe the bug**
An argument cotaining spaces(ex. "1 2") passed to an alias is split.

**To Reproduce**
1. ./pw --init pdm
2. add an alias
```toml:pyproject.toml
[tool.pyprojectx]
pdm = "pdm"
[tool.pyprojectx.aliases]
run = "pdm run"
[tool.pdm]
[project]
```
3. create run.py
```
import sys
print(sys.argv)
```
4. run scripts with alias or tool
```sh
$ ./pw pdm run run.py '1 2'
['run.py', '1 2']
$ ./pw run run.py '1 2'
['run.py', '1', '2']
```

**Expected behavior**
Not be split.

**Desktop (please complete the following information):**
 - Python  version: 3.10
 - OS: Ubuntu-22.04 (WSL)

```sh
$ ./pw --version
0.9.9
$ ./pw pdm --version
PDM, version 2.4.7
```

**Comment by houbie** [comment]
Thany for the feedback. Will look into this next week.

**Comment by houbie** [comment]
fixed in 1.0.0b1
